### ISSUE TITLE
[Feature Request] prepare-second-page

### ISSUE THREAD

**Issue by L483** [IKE-B/voice-revive#20]
### Problem/Motivation

A second screen for finge-granular settings is needed so that the home screen does not become bloated.

### Solution Proposal

Add a second screen with stubs of sliders for potential value configurations.
The screen should be reachable via a navigation item and a drawer.

### Impact

Code needs to be refactored. The voice changing feature must be extracted into a globally available component, so that it can be continuously controlled across all screens and that its state is synchronized across all screens.

This feature relies on code changes introduced in #16 - it branched off of that branch before #16 was merged into dev. This feature should be merged into dev after #16.

### Acceptance Criteria

- [x] A new screen with slider stubs exists
- [x] Navigation between screens is possible
- [x] Voice changing can be controlled globally
- [x] Documented everything

### Further Information

BLoCs will be very useful for creating and managing global state.

### Related Issues

- #16
